WooCommerce/Classes/Authentication/Prologue/LoginPrologueViewController.swift

```diff
@@ -88,7 +90,9 @@ private extension LoginPrologueViewController {
     /// This is contained in a child view so that this view's background doesn't scroll.
     ///
     func setupCarousel(isNewToWooCommerceButtonShown: Bool) {
-        let carousel = LoginProloguePageViewController()
+        let pageTypes: [LoginProloguePageType] = isOnboardingFeatureEnabled ?
+        [.getStarted]: [.stats, .orderManagement, .products, .reviews]
```

Comment:
Super nit: 
1. Consider adding a space after `:` to differentiate this from the normal usage of `:`.
2. Removing the new line would make this more readable I guess.

Reply:
I thought I'd get a warning for the line to be too long but it's actually fine 😄 updated in https://github.com/woocommerce/woocommerce-ios/pull/7291/commits/c7dd9ab4d249c7c36cd770a2275570c43fa9f76d